crypto/evp/evp_fetch.c

```diff
@@ -306,7 +306,9 @@ void *evp_generic_fetch(OPENSSL_CTX *libctx, int operation_id,
 #else
         ERR_raise_data(ERR_LIB_EVP, code,
                        "%s, Algorithm (%s), Properties (%s)",
-                       (openssl_ctx_is_default(libctx)
+                       (openssl_ctx_is_global_default(libctx)
+                        ? "Global default library context"
+                        : openssl_ctx_is_default(libctx)
                         ? "Default library context"
```

Comment:
Perhaps the string should be something like "Thread Local default library context" to more clearly distinguish it from the global default.

Reply:
That's sensible.  Done

Reply:
I ended up refactoring a bit more so there would be only one place where these strings are defined.  It's currently static in this file, but could easily be moved to a libcrypto-internal function if need be (subject for another PR, meybe?)